src/styles/blocks/header.scss

```diff
@@ -21,15 +23,22 @@
     padding: 0;
   }
 
+
   &__item {
     position: relative;
     color: #060b35;
     list-style: none;
     line-height: 60px;
+    transition: color $animationDuration;
+
+    &:hover {
+      color: #00acdc;
+    }
 
     &:not(:last-child) {
       margin-right: 20px;
     }
+
```

Comment:
this line is redundant
also, only one line should be between selectors or selector's rules
FYI